### PR TITLE
updating the bundle template readme

### PR DESCRIPTION
**Pull request by WillBeebe**
This gets us in-sync with the repo manager but see my comments below on sections that might not be relevant for everyone using the CLI.

resolves: https://github.com/massdriver-cloud/massdriver/issues/1423

### CODE REVIEW

File: pkg/bundle/templates/terraform/README.md [hunk 1]
```diff
@@ -1,54 +1,183 @@
...
+
+Our bundles aren't intended to be used locally, outside of testing. Instead, our bundles are designed to be configured, connected, deployed and monitored in the [Massdriver][website] platform.
+
+### What are Bundles?
+
+Bundles are the basic building blocks of infrastructure, applications, and architectures in [Massdriver][website]. Read more [here](https://docs.massdriver.cloud/concepts/bundles).
+
+## Bundle
+
+<!-- COMPLIANCE:START -->
```
Comment (WillBeebe) [inline pkg/bundle/templates/terraform/README.md:35]: these links are Massdriver-heavy. We want this for our bundles but not for 3rd-party.

File: pkg/bundle/templates/terraform/README.md [hunk 2]
```diff
@@ -1,54 +1,183 @@
...
 
-Two development `tfvars` files are provided for setting `params` and `connections`:
+[logo]: https://raw.githubusercontent.com/massdriver-cloud/docs/main/static/img/logo-with-logotype-horizontal-400x110.svg
 
-```shell
-cd src
-terraform init
-terraform plan -var-file=./dev.connections.tfvars.json -var-file=./dev.params.tfvars.json
-```
+[docs]: <md printf $utm_link "https://docs.massdriver.cloud/" "docs" md>
```
Comment (WillBeebe) [inline pkg/bundle/templates/terraform/README.md:146]: These are also massdriver-specific. We might have to trim these from the bundle generator and overwrite them w/ the repo manager.

File: pkg/bundle/templates/terraform/README.md (hunk 1 above)
Comment (xpositivityx) [reply pkg/bundle/templates/terraform/README.md:35]: does the CLI take that in to account?

Comment (coryodaniel) [reply pkg/bundle/templates/terraform/README.md:35]: I think these are fine to leave for now. As a part of this [ticket](https://github.com/massdriver-cloud/massdriver-cli/issues/114) we’ll be moving the name Liquid bundle template to a GitHub repo and we’ll be adding the ability to do custom prompts #109 fir fields like GitHub org and the types you added in the other changes file here. 

We should get the precommit updated here as well w @jaketf’s JSON Schema doc gen

Comment (WillBeebe) [reply pkg/bundle/templates/terraform/README.md:35]: Sounds good 👍 

The precommit was updated and merged here: https://github.com/massdriver-cloud/massdriver-cli/pull/128

Comment (WillBeebe) [reply pkg/bundle/templates/terraform/README.md:35]: @xpositivityx the CLI has no notion of MD or not atm

Comment (xpositivityx) [reply pkg/bundle/templates/terraform/README.md:35]: Cory's comment cleared that up.

**Review by xpositivityx — APPROVED** [review]

**Pull request merged**